evaluation/README.md

```diff
@@ -1 +1,87 @@
-# Evaluate your app
+# Evaluate Your AI App
+
+## Overall
+
+Evaluation is a critical process to measure the performance, quality and effectiveness of AI apps. This guide will help you to evaluate your AI apps and view the results in AI Toolkit.
+
+## Run evaluation
+### Step 1: Set up the Python environment
+
+1. Ensure Python (with version equal to or greater than 3.8) is installed.
+1. Install required libraries with the following command under project root:
+   ```bash
+   pip install -r ai-resource/evaluation/requirements.txt
+   ```
+
+### Step 2: Import your AI app as the evaluation target
+
+Import your AI app in the script and then wrap it into evaluation target. The evaluation target is a function accepts question and return an object of answer.
+
+```python
+def target_app(question):
+   # import your AI app and wrap it into the evaluation target, like:
+   response = my_app(question)
+   return { "answer": response.message }
+```
+
+### Step 3: Set up LLM for AI-assisted evaluator
+
+Most of the built-in evaluators are AI-assisted:
+- GroundednessEvaluator
+- RelevanceEvaluator
+- CoherenceEvaluator
+- FluencyEvaluator
+- SimilarityEvaluator
+
+If one of those evaluators is selected, you need to set `model_config` with Azure OpenAI model.
+
+> **Note**: `model_config` with OpenAI configuration is not supported for `promptflow-eval` SDK yet and it's planed to be supported in version `v0.3.3`.
+>
+
+**Use Azure OpenAI Endpoint**:
+1. Create an Azure OpenAI resource and deploy a model. Detailed instructions can be found [here](https://learn.microsoft.com/azure/ai-services/openai/how-to/create-resource).
```

Comment:
Create an Azure OpenAI resource and deploy a model. Detailed instructions can be found in the [Azure OpenAI documentation] (https://learn.microsoft.com/azure/ai-services/openai/how-to/create-resource).